monitoring/rid_qualifier/run_locally.sh

```diff
@@ -0,0 +1,27 @@
+
+#!/usr/bin/env bash
+
+AUTH='--auth=NoAuth()'
+# NB: A prerequisite to run this command locally is to have a running local DSS instance session via '/build/dev/run_locally.sh', for more information see https://github.com/interuss/dss/blob/master/build/dev/standalone_instance.md
+
+LOCALE='--locale=che'
+
+INJECTION_URL='--injection_base_url=http://localhost:8070'
+
+INJECTION_SUFFIX='--injection_suffix=/sp/uss1/tests/9a20678b-fad4-49e6-9009-b4891aa77cb7'
```

Comment:
The base URL of the injection API provided by the virtual USS "uss1" in the rid_qualifier mock is `http://localhost:8070/sp/uss1`, and that should be a primary input to rid_qualifier.  Basically, we want to answer the question, "Where should test data be injected for this USS?" which is equivalent to "Where is this USS's implementation of the injection API located?"  The answer to both (in the case of the rid_qualifier mock) is `http://localhost:8070/sp/uss1`.

The user shouldn't be providing the test ID; rid_qualifier should generate that as appropriate, and we will probably want more than one injected test at some point.

So, no need for separate `INJECTION_URL` and `INJECTION_SUFFIX`; there should just be a single `INJECTION_BASE_URL` (also fine to call it just `INJECTION_URL`).

Reply:
Yes agreed as per our conversation, I am removing the `INJECTION_SUFFIX`.